extensions/amp-user-notification/0.1/amp-user-notification.js

```diff
@@ -126,6 +125,11 @@ export class AmpUserNotification extends AMP.BaseElement {
       assertHttpsUrl(this.dismissHref_, this.element);
     }
 
+    const alwaysShowAttr = this.element.getAttribute('always-show');
+    /** @private @const {boolean} */
+    this.alwaysShow_ = (this.element.hasAttribute('always-show') &&
+        alwaysShowAttr != '0' && alwaysShowAttr != 'false');
```

Comment:
yeah that seems more consistent and saner. i feel like we dont really need a value though.

/cc @rudygalfi @dvoytenko